content/guides/getting-started/styling-components.md

```diff
@@ -93,10 +94,10 @@ Overlays + Modals are usually really good.
 This guide will help you setup your test infrastructure to render your
 component's styles properly. Solving certain runtime errors, such as those with
 ThemeProviders or Material UI, is covered on the
-"[Getting Components to Work](/guides/getting-started/getting-components-to-work)"
+[Getting Components to Work](/guides/getting-started/getting-components-to-work)
```

Comment:
[Getting Components to Work ](https://deploy-preview-4186--cypress-docs.netlify.app/guides/getting-started/getting-components-to-work) page is incomplete:
<img width="1122" alt="Screen Shot 2022-05-19 at 3 46 48 PM" src="https://user-images.githubusercontent.com/26726429/169377084-6a7e3811-8ad4-48cc-b1b6-cbbf08b531ce.png">

Reply:
will look into if this link and page is needed